### PR TITLE
Update version number and changelog for ROCm 6.1 (#169)

### PR DESCRIPTION
**Pull request by samjwu**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/share/rocmcmakebuildtool
 include(ROCMCreatePackage)
 include(ROCMSetupVersion)
 
-rocm_setup_version(VERSION 0.11.0)
+rocm_setup_version(VERSION 0.12.0)
```
Comment (peterjunpark) [inline CMakeLists.txt]: Should this be 0.13.0? That's what the changelog says.

The version is 0.12.0 in the 6.1 release branch https://github.com/ROCm/rocm-cmake/blob/release-staging/rocm-rel-6.1/CMakeLists.txt

Comment (samjwu) [reply CMakeLists.txt]: created https://github.com/ROCm/rocm-cmake/pull/210 to address this. will cherry pick that change into this PR if it is merged

**Review by peterjunpark — APPROVED** [review]
lgtm

**Review by cgmb — APPROVED** [review]

**Comment by cgmb** [comment]
Root cause was that I couldn't merge #171 until after branching due to test failures... or, at least, one of the root causes.

**Pull request merged**